src/templates/renderServiceTemplates.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const dots = require('dot');
+const crypto = require('crypto');
+
+/**
+ * Render templates for services
+ * @param {Config} config
+ * @returns {Promise<void>}
+ */
+async function renderServiceTemplates(config) {
+  const rpcpassword = crypto.randomBytes(12).toString('base64');
+  try {
+    const files = fs.readdirSync('./templates');
+    console.log(config);
+    for (const file of files) {
+      console.log(file);
+      const fileContents = fs.readFileSync('./templates/' + file, 'utf-8');
+      const fileTemplate = dots.template(fileContents);
+      const fileOutput = fileTemplate({config, rpcpassword});
+      console.log(fileOutput);
```

Comment:
I guess this function should respond with a list of files and their content (Object.<string, string>). After that, we need one more function which accepts this list and create/update volume(s?) with these files.

Reply:
OK, I tried to implement another function. I am 100% certain I have done the dependency injection wrong. I've tried for days, no matter what I do I just cannot get awilix to work. All of the code executes perfectly and the file is templated and written to disk, but I see errors like `renderServiceTemplates is not a function` or some Awilix injection error no matter what way I try it.